### PR TITLE
Move uninstall command to new subctl implementation

### PR DESCRIPTION
**Pull request by Jaanki**
Signed-off-by: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

**Review by tpantelis — APPROVED** [review]

File: cmd/subctl/uninstall.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	"github.com/submariner-io/submariner-operator/internal/constants"
+	"github.com/submariner-io/submariner-operator/internal/exit"
+	"github.com/submariner-io/submariner-operator/pkg/client"
+	"github.com/submariner-io/submariner-operator/pkg/uninstall"
+)
+
+var (
+	noPrompt  bool
+	namespace string
+)
```
Comment (tpantelis) [inline cmd/subctl/uninstall.go]: To keep these separate from other command vars, let's encapsulate them into a struct:

```suggestion
var uninstallOptions struct {
	noPrompt bool
	namespace string
}
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: cmd/subctl/uninstall.go (hunk 1 above)
Comment (Jaanki) [reply cmd/subctl/uninstall.go]: Done

**Review by tpantelis — APPROVED** [review]

**Comment by dfarrell07** [comment]
If this "moves" the command, shouldn't it be removed from where it was before? Is that planned for later?

**Review by dfarrell07 — APPROVED** [review]

**Comment by tpantelis** [comment]
> If this "moves" the command, shouldn't it be removed from where it was before? Is that planned for later?

yeah it will be once the rest of the commands are "moved".

**Pull request merged**